stress/common/src/main/java/alluxio/stress/fuse/FuseIOParameters.java

```diff
@@ -25,7 +25,8 @@
   public static final String FIELD_READ_RANDOM = "mReadRandom";
```

Comment:
remove the read random parameters?

Reply:
I'm going to have a separate PR for removing all random read stuff.